runtime/runtime-params-estimator/src/lib.rs

```diff
@@ -668,46 +668,65 @@ fn action_function_call_base(ctx: &mut EstimatorContext) -> GasCost {
     total_cost.saturating_sub(&base_cost, &NonNegativeTolerance::PER_MILLE)
 }
 fn action_function_call_per_byte(ctx: &mut EstimatorContext) -> GasCost {
```

Comment:
So here we vary only the size of the argument, not the size of a method name. Could you sanity-check that long method names are not problematic (not sure if theer's some small size limit there).

Reply:
There is this parameter: `max_length_method_name: 256`

Quick sanity checks suggest that there is no measurable difference between different method lengths in the range 1-255. (Mostly because the variance is just too high.)